app/com/arpnetworking/metrics/portal/alerts/impl/NoAlertRepository.java

```diff
@@ -68,7 +69,7 @@ public void close() {
      * {@inheritDoc}
      */
     @Override
-    public Optional<Alert> get(final UUID identifier) {
+    public Optional<Alert> get(final UUID identifier, final Organization organization) {
         assertIsOpen();
         LOGGER.debug()
                 .setMessage("Getting alert")
```

Comment:
I think this is probably the last item. Add organization to all the relevant log statements.